src/Manager.sol

```diff
@@ -673,7 +683,10 @@ abstract contract Manager is
         if (_isAllZeros(siblingContract)) {
             revert InvalidSiblingZeroBytes();
         }
+
         _getSiblingsStorage()[chainId_] = siblingContract;
+
+        emit SiblingUpdated(chainId_, siblingContract);
```

Comment:
Should we include the old sibling contract address there too?
If unset, it'd be zero -- just whatever was in the mapping before we set the new value.